opaque_keys/edx/tests/test_util.py

```diff
@@ -0,0 +1,76 @@
+"""Test the opaque key utility functions."""
+
+from unittest import TestCase
+
+from opaque_keys.edx.locator import CourseLocator
+from opaque_keys.edx.util import get_filename_safe_course_id
+
+
+VALID_COURSE_ID = u'{}'.format(CourseLocator(org='org', course='course_id', run='course_run'))
```

Comment:
Nit: Rather than just generating this programmatically, could we test specifically both a dynamically generated version and a static version? That way, in future, if tests fail, we can specifically peer into whether we're seeing failure on the scheme that exists now, as of August 2016, or due to a change in how `CourseLocators` are structured.